ape_solidity/compiler.py

```diff
@@ -423,6 +423,11 @@ def _get_pragma_spec(path: Path) -> Optional[NpmSpec]:
         # Build map of pragma-specs.
         source_by_pragma_spec = {p: _get_pragma_spec(p) for p in source_paths_to_compile}
 
+        # If not Solidity version has been auto-installed while fetching the
+        # contract versions, install the latest
+        if self.installed_versions:
```

Comment:
should this be `if not self.installed_versions:` ?

Reply:
Dang it, I had that but forgot to save before pushing 😂

Reply:
Just force pushed to correct